sdcm/tester.py

```diff
@@ -426,9 +426,11 @@ def argus_get_scylla_version(self):
             self.log.info("Collection Scylla version for argus...")
             version_regex = re.compile(r'(([\w.~]+)-(0.)?([0-9]{8,8}).(\w+).)')
             version_str = self.db_cluster.nodes[0].get_scylla_binary_version()
-            if (match := version_regex.match(version_str)):
+            if version_str and (match := version_regex.match(version_str)):
                 self.argus_test_run.run_info.details.scylla_version = match.group(2)
                 self.argus_test_run.save()
+            else:
```

Comment:
Why only warning?
I think we want to fail the test and check why we couldn’t get the version.

Reply:
Done